src/client/src/containers/NewProject/styles.module.css

```diff
@@ -46,3 +46,9 @@
 .quickStartContainer {
   margin-top: 1em;
 }
+
+@media (max-width: 1120px) {
+  .container {
+    max-width: 100%;
+  }
+}
```

Comment:
This would cause the input elements for `project name` and `save to location` to increase width to 100% on medium screens. Looks like this. Is this desired to be so long?
I would say maybe we could keep this to about 70% for medium screens and then to 100% for small screens.

![image](https://user-images.githubusercontent.com/35667299/62639584-ab7ae600-b8f4-11e9-8049-0c38bb1b2ba5.png)

Reply:
good point, will change it for medium screen sizes